Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,48 @@
+class ClickhouseOdbc < Formula
+  desc "ClickHouse ODBC driver"
+  homepage "https://clickhouse.yandex"
+  url "https://github.com/yandex/clickhouse-odbc.git",
+    :tag => "v1.0.0.20180903",
+    :revision => "17ecca3993e164d2d49360ed9be68becc11b7b05"
+  # head "https://github.com/yandex/clickhouse-odbc.git"
+
+  bottle do
+    # sha256 "256" => :high_sierra
+    # sha256 "256" => :sierra
+    # sha256 "256" => :el_capitan
+  end
+
+  # option "with-debug", "Install debugging version with advanced logs"
```

Comment:
Remove code commented out. No debug option.

Reply:
this from audit too. maybe uncomment?